### DISCUSSION TITLE
Expose version number in flake

### DISCUSSION THREAD

**Discussion by srid** [purs-nix/purs-nix#26] [Q&A]
Suppose I reference purs-nix as the following flake input,

```nix
purs-nix.url = "github:purs-nix/purs-nix/ps-0.14";
```

Is there a way to dynamically refer to the version number, 0.14, by inspecting the output of the flake?

Why do I need this? So I can dynamically determine the "for-0_14" in `inputs.purscript-tools.legacyPackages.for-0_14.purescript-language-server`.

**Comment by ursi** [comment]
The only way I can think of to do that is to use
```
(import ./flake.nix).inputs.purs-nix.url
```
and use `substring` on it to get the version information.

**Comment by ursi** [comment]
If this is something people want to do often. I suppose it should be exposed on the purs-nix flake somewhere.
